docs/source/tutorials/configuration.rst

```diff
@@ -81,7 +81,7 @@ Usually, manim is ran from the command line by executing
 
 .. code-block:: bash
 
-   $ manim <file.py> SceneName
+   $ manim [OPTIONS] [FILE] [SCENES]
```

Comment:
I think how it previously made sense than this?
Read the below para.

Reply:
Sure, I'll change this back.